cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/ResourceDistributionGoal.java

```diff
@@ -49,8 +49,6 @@
   // Flag to indicate whether the self healing failed to relocate all replicas away from dead brokers in its initial
   // attempt and currently omitting the resource balance limit to relocate remaining replicas.
   private boolean _selfHealingDeadBrokersOnly;
-  private Set<Integer> _brokerIdsAboveBalanceUpperLimit;
```

Comment:
PR https://github.com/linkedin/cruise-control/pull/95 has a different fix for the same issue.
I think this one looks good, too!